examples/quantum_annealing/price_optimization/QUBO-Pricing.ipynb

```diff
@@ -0,0 +1,2855 @@
+      "rev_std: 810.9564396486212\n",
+      "penalty_term: -0.006314725265838206\n",
+      "---\n",
+      "beta_sample:10000.0\n",
+      "max_revenue:11668.057902789285\n",
+      "prediction_variance:702.114905983479\n",
+      "energy:7009481.0\n",
+      "opt_demand:[193.16540581130278, 189.5001278388014, 188.47908467385338, 199.47384504135448, 192.31622092125497, 192.94709710762896, 172.9586102534359]\n",
+      "opt_prices:[10, 8, 8, 5, 10, 8, 13]\n",
+      "rev_std: 620.5042342568443\n",
+      "penalty_term: -0.0019320007413625717\n",
+      "---\n",
+      "beta_sample:100000.0\n",
+      "max_revenue:12133.533110080407\n",
+      "prediction_variance:702.0571376283112\n",
+      "energy:70193580.234375\n",
+      "opt_demand:[193.16540581130278, 175.06183547909168, 168.32494666836413, 163.0137226806254, 169.2156327071633, 176.86116017646032, 168.26133247161616]\n",
+      "opt_prices:[10, 12, 10, 10, 8, 8, 12]\n",
+      "rev_std: 700.4772829770868\n",
+      "penalty_term: 0.004653960466384888\n",
+      "---\n",
+      "beta_sample:1000000.0\n",
+      "max_revenue:10843.245671844892\n",
+      "prediction_variance:701.7569277253392\n",
+      "energy:701746084.4921875\n",
+      "opt_demand:[193.16540581130278, 182.28098165894653, 182.0115887610362, 175.45359909257996, 179.27732026268376, 182.717054502723, 196.43961960101143]\n",
+      "opt_prices:[10, 10, 8, 10, 8, 8, 5]\n",
+      "rev_std: 590.4245277513222\n",
+      "penalty_term: 0.012520194053649902\n",
+      "---\n",
+      "beta_sample:10000000.0\n",
+      "max_revenue:11696.71176176828\n",
+      "prediction_variance:702.2911264607972\n",
+      "energy:7022899567.890625\n",
+      "opt_demand:[200.38455199115765, 188.74847757176371, 173.54567690046386, 166.60791432332866, 154.66085417457893, 173.57685606412835, 173.3708773666939]\n",
+      "opt_prices:[8, 10, 12, 10, 12, 5, 10]\n",
+      "rev_std: 670.533082027326\n",
+      "penalty_term: -0.005584716796875\n",
+      "---\n",
+      "beta_sample:100000000.0\n",
+      "max_revenue:11259.819536849312\n",
+      "prediction_variance:701.8711813398561\n",
+      "energy:70187106874.17188\n",
+      "opt_demand:[193.16540581130278, 189.5001278388014, 181.2599384939985, 174.95848367890017, 177.39424681992944, 191.22305398600184, 187.4710989807441]\n",
+      "opt_prices:[10, 8, 10, 10, 8, 5, 10]\n",
+      "rev_std: 610.443345015263\n",
+      "penalty_term: 0.00579833984375\n",
+      "---\n",
+      "beta_sample:1000000000.0\n",
+      "max_revenue:12094.573836154155\n",
+      "prediction_variance:702.2972898129863\n",
+      "energy:702297277718.4141\n",
+      "opt_demand:[193.16540581130278, 189.5001278388014, 170.43121922421625, 172.4763859895292, 174.9031719840405, 165.79446227107266, 166.27428943744752]\n",
+      "opt_prices:[10, 8, 13, 8, 8, 12, 10]\n",
+      "rev_std: 690.5026388893243\n",
+      "penalty_term: 0.001708984375\n"
+     ]
+    }
+   ],
+   "source": [
```

Comment:
The whitespace seems to have gotten messed up somehow. Please paste in this, which should work instantly with the standard Command/CTRL -> / for commenting/uncommenting multi lines. 
```
# beta_options=[1e2,1e3,1e4,1e5,1e6,1e7,1e8,1e9]
# results = {beta_option:[] for beta_option in beta_options}
# beta_samples = beta_options*6

# for beta_i in beta_samples:
#     max_revenue, prediction_variance, energy, opt_demand, opt_prices, rev_std = optimize(
#        a = a,
#        b = b,
#        data_x = data_x,
#        selected_hist_prices = p_data,
#        price_levels = price_levels,
#        Lp = Lp,
#        Ld = Ld,
#        sigma = sigma ,
#        beta=beta_i,
#        vol_bound=None,
#    )
#     print('---')
#     print(f"beta_sample:{beta_i}")
#     print(f"max_revenue:{max_revenue}")
#     print(f"prediction_variance:{prediction_variance}")
#     print(f"energy:{energy}")
#     print(f"opt_demand:{opt_demand}")
#     print(f"opt_prices:{opt_prices}")
#     print(f"rev_std: {rev_std}")
#     print(f"penalty_term: {max_revenue-beta_i*prediction_variance+energy}")
#     results[beta_i].append([energy, max_revenue, prediction_variance, rev_std])
```